update/mock/epochStartNotifierStub.go

```diff
@@ -0,0 +1,81 @@
+package mock
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/epochStart"
+)
+
+// EpochStartNotifierStub -
+type EpochStartNotifierStub struct {
+	RegisterHandlerCalled                 func(handler epochStart.ActionHandler)
+	UnregisterHandlerCalled               func(handler epochStart.ActionHandler)
+	NotifyAllCalled                       func(hdr data.HeaderHandler)
+	NotifyAllPrepareCalled                func(hdr data.HeaderHandler, body data.BodyHandler)
+	epochStartHdls                        []epochStart.ActionHandler
+	NotifyEpochChangeConfirmedCalled      func(epoch uint32)
+	RegisterForEpochChangeConfirmedCalled func(handler func(epoch uint32))
+}
+
+// RegisterForEpochChangeConfirmed -
+func (esnm *EpochStartNotifierStub) RegisterForEpochChangeConfirmed(handler func(epoch uint32)) {
+	if esnm.RegisterForEpochChangeConfirmedCalled != nil {
+		esnm.RegisterForEpochChangeConfirmedCalled(handler)
+	}
+}
+
+// NotifyEpochChangeConfirmed -
+func (esnm *EpochStartNotifierStub) NotifyEpochChangeConfirmed(epoch uint32) {
+	if esnm.NotifyEpochChangeConfirmedCalled != nil {
+		esnm.NotifyEpochChangeConfirmedCalled(epoch)
+	}
+}
+
+// RegisterHandler -
+func (esnm *EpochStartNotifierStub) RegisterHandler(handler epochStart.ActionHandler) {
+	if esnm.RegisterHandlerCalled != nil {
+		esnm.RegisterHandlerCalled(handler)
```

Comment:
Forgot to add return here ?

Reply:
no need - this is a mock. somewhat implementation - copied from integration/mock package.